src/main/java/Client/MopeLSPClient.java

```diff
@@ -168,25 +177,28 @@ public void didOpenFile(String path){
     }
 
     /**
-     * This model performs basic checks on a model and returns the number of variables and equations in it.
+     * This method performs basic checks on a model and returns the number of variables and equations in it.
+     * The method checkModel calls the method get() defined in CompletableFuture class and handles the exception by
```

Comment:
This is an implementation detail that should not be in the documentation. Instead, you should specify that this corresponds to https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/scripting_api.html#checkmodel